### DISCUSSION TITLE
sendablechooser help

### DISCUSSION THREAD

**Discussion by Mateo-Johnson** [mjansen4857/pathplanner#536] [Q&A]
In the example sendablechooser code, does it automatically populate itself with my auto options from pathplanner? if not, what is the command to add them as options, because the normal command seems to throw an error.

**Comment by mjansen4857** [comment]
It will auto populate. Need more info on what "normal command" means and what the error is.

**Reply by Mateo-Johnson** [reply]
yeah, so I just took my best guess at trying to manually input options, with this: "autoChooser.addOption("Auto 1", (Command) AutoBuilder.buildAutoChooser());". With this version, there weren't any errors I just wasn't sure if this was the intended way or if it would work

**Reply by mjansen4857** [reply]
`AutoBuilder.buildAutoChooser()` creates the whole SendableChooser, you don't add it to the chooser. So, somewhere near the end of your RobotContainer constructor:
```java
autoChooser = AutoBuilder.buildAutoChooser();
```
